.github/workflows/release.yml

```diff
@@ -10,6 +10,9 @@ jobs:
     name: Release
     if: ${{ github.repository_owner == 'withastro' }}
     runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      id-token: write
```

Comment:
Hmm, looks like this still isn’t the right permissions combo in our case. The workflow failed after merging: https://github.com/withastro/starlight/actions/runs/12316634406/job/34377330918

Might need write access to create the release PR?

Reply:
Ah there was https://github.com/withastro/astro/pull/8752 which updated to `contents: write`. Not much additional context in there though 😁

Reply:
Made a fix PR: https://github.com/withastro/starlight/pull/2665